pkg/provenance/provenance.go

```diff
@@ -73,14 +75,30 @@ func init() {
 	AllProvenanceObjects = make([]ProvenanceOfObject, 0)
 
 }
-
+func onMinikube() bool {
+	ip := os.Getenv("HOST_IP")
+	ipBeginsWith := strings.Split(ip, ".")[0]
+	return ipBeginsWith == "10"
```

Comment:
Minikube IPs also begin with 198. for v0.25 and below.
That should be accommodated here as well.
You can have a OR condition for these two cases.